apps/rehabstod/src/store/errorMiddleware.ts

```diff
@@ -13,17 +13,21 @@ export const errorMiddleware: Middleware =
   (action) => {
     if (isRejectedWithValue(action)) {
       const { method, url } = action.meta.baseQueryMeta.request
-      const { message } = action.payload.data ?? 'No message'
-      const errorMessage = `${message} method '${method}' url '${url}`
-      const { errorCode } = action.payload.data ?? 'No errorCode'
+      let message = 'No message'
+      let errorCode = 'No errorCode'
+      if (action.payload.data) {
+        message = action.payload.data.message ?? 'No message'
+        errorCode = action.payload.data.errorCode ?? 'No errorCode'
+      }
+      const errorMessage = `${message}' method '${method}' url '${url}`
       const errorId = uuidv4()
       const errorData = {
         errorId,
         errorCode,
         message: errorMessage,
         stackTrace: null,
       }
-      dispatch(api.endpoints.logError.initiate({ errorData, ...errorData }))
+      dispatch(api.endpoints.logError.initiate({ ...errorData, errorData }))
```

Comment:
This should be `{ errorData }` without the extra spread. Not sure why typescript is not complaining here.